src/core.jl

```diff
@@ -64,15 +44,11 @@ initialization, and the init work is actually done on that worker to avoid
 unnecessary data copying.
```

Comment:
If we only initialise the Som grid only by the first data sample we are a bit biased aren't we?

Reply:
Yes, this behavior was copied from the intermediate distributed implementation.

Since the SOMs should anyway be initialized totally randomly without _any_ data bias, what if we go for the following:

1. `dstat` to get the distribution of the data
2. generate the SOM using `randn` with the same normal distribution as the data have?

Reply:
that would work. could we also: select workers by random and collect x rows per random worker?

Reply:
yeah, picking 1/N total points from N workers should just work. I'll see what's simpler to write.